Jenkinsfile

```diff
@@ -4,7 +4,7 @@ import groovy.json.JsonOutput
 @Library('realm-ci') _
 repoName = 'realm-js'
 
-platforms = ['win32-ia32', 'win32-x64', 'darwin-x64', 'darwin-arm64', 'linux-x64', 'linux-arm']
+platforms = ['linux-x64']
```

Comment:
It should only be running the legacy test on node.

Reply:
We agreed on this in our CI meeting, as we didn't want to transfer all the legacy tests into GHA.  This will keep at a minimum the tests running on node